app/assets/stylesheets/modules/_uploads.styl

```diff
@@ -155,3 +154,10 @@
 
     .upload-viewer-button
       margin 10px
+
+.view_table
+  border 1px solid #ddd
+
+.element
```

Comment:
This is not a very informative class name. Is there another term that would convey the intended specific use?